### PR TITLE
Create 2021-12-31-medical-diagnosis.md

### PR DESCRIPTION
**Pull request by day0hero**
initial post

### CODE REVIEW

File: _posts/2021-12-31-medical-diagnosis.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+layout: post
+date: 2021-12-31
+title: Medical Diagnosis
+published: true
+tags:
+- medical-diagnosis
+- xray
+- validated-pattern
+---
+Our team recently completed the development of a validated pattern that showcases the capabilities we have at our fingertips when we combine OpenShift and other cutting edge Red Hat technologies to deliver a solution. We've taken an application written completely in Ansible and converted the kubernetes resources into manifests to be deployed, managed and monitored through ArgoCD. Using the validated pattern framework we are able to deploy, manage and integrate with multiple cutting edge Red Hat technologies, and provide a capability that the initial deployment strategy didn't have available to it: a lifecycle. Everything you need to take this pattern for a spin is in [git](https://github.com/hybrid-cloud-patterns/medical-diagnosis).
```
Comment (beekhof) [inline _posts/2021-12-31-medical-diagnosis.md:12]:
```suggestion
Our team recently completed the development of a validated pattern that showcases the capabilities we have at our fingertips when we combine OpenShift and other cutting edge Red Hat technologies to deliver a solution. 

We've taken an application defined imperatively in an Ansible playbook and converted it into GitOps style declarative kubernetes resources. Using the validated pattern framework we are able to deploy, manage and integrate with multiple cutting edge Red Hat technologies, and provide a capability that the initial deployment strategy didn't have available to it: a lifecycle. Everything you need to take this pattern for a spin is in [git](https://github.com/hybrid-cloud-patterns/medical-diagnosis).
```

File: _posts/2021-12-31-medical-diagnosis.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+#### Pattern Deployment
+---
+Before getting started, you need to install the following packages and collections.
+**PREREQUISITES** <br>
+
+| Packages |
+|:--------:|
+| git |
+| make |
+| ansible |
```
Comment (beekhof) [inline _posts/2021-12-31-medical-diagnosis.md:34]: helm? oc?

Comment (beekhof) [reply _posts/2021-12-31-medical-diagnosis.md:34]: Actually, let's not even put any of this here.  Lets just link to the install instructions

Comment (day0hero) [reply _posts/2021-12-31-medical-diagnosis.md:34]: @beekhof - i actually spun up a fresh vm so I could walk all the way through ( to catch assumptions ) and I hit this with oc and helm ... will link to install instructions

**Comment by day0hero** [comment]
@beekhof poke

**Pull request merged**